### DISCUSSION TITLE
Pan in a specific direction

### DISCUSSION THREAD

**Discussion by alexkutaladze** [software-mansion/react-native-gesture-handler#1318] [Q&A] [answered]
Hi,
Is there a way to implement pan gesture handler so that the user is only able to pan down vertically?

**Comment by mrousavy** [comment, marked as answer]
Yes, you can use [`failOffsetY`](https://docs.swmansion.com/react-native-gesture-handler/docs/handler-pan#failoffsety) and [`activeOffsetY`](https://docs.swmansion.com/react-native-gesture-handler/docs/handler-pan#activeoffsety) to specify your handler's activation criteria.
